src/com/lilithsthrone/utils/Colour.java

```diff
@@ -338,9 +338,13 @@
 	// Body parts:
 
 	// Skin (Human and Demon):
+	SKIN_PORCELAIN(Util.newColour(0xDBCDB9), Util.newColour(0xDBCDB9), "porcelain")
```

Comment:
this missing a comma too (so I guess all the rows in this section that you added)